### ISSUE TITLE
REST API and peer

### ISSUE THREAD

**Issue by byteschain** [hyperledger-archives/fabric#2141]
## Description

I want to know why REST API  is not separated from peer

**Comment by GrapeBaBa** [comment]
that is not true, only develop environment enable rest service in peer, for production rest service should run in the NVP.

**Comment by byteschain** [comment]
I didn't find NVP

**Comment by mingjinc** [comment]
@GrapeBaBa How could we set up the production environment? I only see the doc about dev environment setup.

**Comment by GrapeBaBa** [comment]
https://github.com/hyperledger/fabric/wiki/Client-SDK-Specification, this doc introduce the NVP, but in future it will change per this doc.
